api/envoy/config/common/accesslog/v3/common.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto3";
+
+package envoy.config.common.accesslog.v3;
+
+import "udpa/annotations/status.proto";
+import "udpa/annotations/versioning.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.config.common.accesslog.v3";
+option java_outer_classname = "CommonProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// Common configuration for all accesslog filter.
```

Comment:
This needs a title annotation to pass the docs generation: e.g. `[#protodoc-title: title-goes-here]`

Reply:
Thanks..Fixed.